### PR TITLE
FISH-5748 Alter OpenID Connect Doc

### PR DESCRIPTION
**Pull request by kalinchan**
Tested what variables I can use.

## Can be used
* MP Config Properties (`${MPCONFIG=payara.security.openid.clientId`)
* Expression Language (`#{testBean.clientId}`)

## Can't be used
* Password Aliases 
* Environment Variables / System Properties

### CODE REVIEW

**Comment by rdebusscher** [comment]
Environment Variables and System Properties can be used indirectly through MicroProfile Config

**Comment by kalinchan** [comment]
I have added a note to reflect this.

**Comment by kalinchan** [comment]
This has been sat in review for a while, unless there are any objections, I will merge this PR in 48 hours.

File: docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc [hunk 1]
```diff
@@ -487,10 +487,10 @@ When fields, which are lists (e.g. `scopesSupported`, `responseTypesSupported`),
 
 The client secret can be input directly, or for added security it can be aliased using any of the following features:
```
Comment (aubi) [inline docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc:488]: "using any of the following features"

There is only one remaining :-) And I would put MicroProfile into the link, so it's more clear.

File: docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc [hunk 2]
```diff
@@ -487,10 +487,10 @@ When fields, which are lists (e.g. `scopesSupported`, `responseTypesSupported`),
...
 
-- xref:/documentation/payara-server/password-aliases/README.adoc[Password Aliases]
-- xref:/documentation/payara-server/server-configuration/var-substitution/README.adoc[Environment Variables / System Properties]
 - xref:/documentation/microprofile/config/README.adoc[Config API]
 
+NOTE: Environment Variables and System Properties can be used indirectly through MicroProfile Config.
```
Comment (aubi) [inline docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc:492]: This is what the previous text said already?

**Review by aubi — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc (hunk 2 above)
Comment (kalinchan) [reply docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc:490]: Environment Variables and System Properties cannot be used via variable substitution, if a user wants to use them, they will have to use them indirectly through MicroProfile Config

**Review by aubi — APPROVED** [review]

**Pull request merged**